src/backends/caffe/caffelib.cc

```diff
@@ -2641,21 +2641,51 @@ namespace dd
                     else if (inputc._timeserie)
                       {
                         std::vector<double> target;
+                        std::vector<double> target_unscaled;
+                        CSVTSCaffeInputFileConn *cic
+                            = reinterpret_cast<CSVTSCaffeInputFileConn *>(
+                                &inputc);
                         for (size_t k = 0; k < dv_float_data.at(j).size(); k++)
                           {
-                            target.push_back(dv_float_data.at(j).at(k));
+                            float res = dv_float_data.at(j).at(k);
+                            target.push_back(res);
+                            if (!cic->_dont_scale_labels)
+                              {
+                                double max
+                                    = cic->_max_vals[cic->_label_pos[k]];
+                                double min
+                                    = cic->_min_vals[cic->_label_pos[k]];
+                                if (cic->_scale_between_minus1_and_1)
+                                  res += 0.5;
```

Comment:
Is this correct? if values are scaled between -1 and 1, then this should be `res = (res+1) * 0.5`, I think?

Reply:
indeed we are doing the exact opposite of https://github.com/jolibrain/deepdetect/blob/9597bc0aa36b8d5a397d0b98937cb54347035e50/src/csvinputfileconn.h#L338
and i think you are right, this flag should be _scale_between_minus0.5_and_0.5 ...

Reply:
changed the name of this flag => done